Mathlib/Algebra/GroupWithZero/Units/Basic.lean

```diff
@@ -72,8 +72,7 @@ theorem not_isUnit_zero [Nontrivial M₀] : ¬IsUnit (0 : M₀) :=
 
 namespace Ring
 
-open scoped Classical
-
+open Classical in
```

Comment:
Does `open scoped Classical` also work? Would you like to use that instead? (Also applies in ~30 other places.)

Reply:
Yes, it works, but doesn't make a difference and is a little bit longer.

Reply:
Fair enough; I won't insist on that one.